src/LanguageServer/Impl/Sources/DefinitionSource.cs

```diff
@@ -98,26 +112,41 @@ internal sealed class DefinitionSource {
                         if (member is IPythonInstance v) {
                             return new Reference { range = v.Location.Span, uri = v.Location.DocumentUri };
                         }
-                        return await FromMemberAsync(member, null, eval, cancellationToken);
+                        return await FromMemberAsync(member, null, statement, analysis, cancellationToken);
                     }
             }
 
-            if (node != null && CanNavigateToModule(module) && module is IDocument doc) {
+            module = ToRealModule(module, analysis);
```

Comment:
Does it mean we can get here from stub?

Reply:
It seems to work but more reliable is to set real module to stub when created. Do you prefer me to so that?

Reply:
Made relationship explicit